app-modules/audit/src/Filament/Actions/AuditAttachAction.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Assist\Audit\Filament\Actions;
+
+use OwenIt\Auditing\Contracts\Auditable;
+use Filament\Tables\Actions\AttachAction;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class AuditAttachAction extends AttachAction
+{
+    protected function setUp(): void
```

Comment:
Are we planning to use the custom Attach and Detach actions? Or is this covered by the other work done in setting this up behind the scenes?

Reply:
This was covered by the behind-the-scenes work. I just decide to keep them around for a little bit just in case we end up needing them. As this behind-the-scenes work is still a little green feeling to me.